lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -121,18 +121,27 @@ tokenize(String, Line, Column, Opts) ->
 
   Existing = case lists:keyfind(existing_atoms_only, 1, Opts) of
     {existing_atoms_only, true} -> true;
+    {existing_atoms_only, false} -> false;
```

Comment:
@michalmuskala, thank you for the suggestion! I've tried your solution but unfortunately, it gives me an unsafe variable error. I'm going with @josevalim's suggestion because it can cover the rest of the unwanted cases as well.